extensions/common/context.cc

```diff
@@ -422,15 +423,26 @@ std::string_view nodeInfoSchema() {
 void populateExtendedHTTPRequestInfo(RequestInfo* request_info) {
   populateExtendedRequestInfo(request_info);
 
-  getValue({"request", "referer"}, &request_info->referer);
-  getValue({"request", "useragent"}, &request_info->user_agent);
-  getValue({"request", "id"}, &request_info->request_id);
+  if (getValue({"request", "referer"}, &request_info->referer)) {
+    sanitizeBytes(&request_info->referer);
+  }
+  if (getValue({"request", "useragent"}, &request_info->user_agent)) {
+    sanitizeBytes(&request_info->user_agent);
+  }
+  if (getValue({"request", "id"}, &request_info->request_id)) {
+    sanitizeBytes(&request_info->request_id);
+  }
   std::string trace_sampled;
   if (getValue({"request", "headers", "x-b3-sampled"}, &trace_sampled) &&
```

Comment:
Should `x-b3-sampled` be sanitized as well?

Reply:
No, since this just compares against "1".